docs/reference/integrations/optimizely.md

```diff
@@ -1,18 +1,24 @@
-import Tabs from '@theme/Tabs';
-import TabItem from '@theme/TabItem';
+# Optimizely
 
-# Create assignment table from Optimizely
+In order to perform its analyses, Eppo needs access to an assignment table in your data warehouse that lists each user that comes through the system and which variant they saw at which time.
 
-_Note:_ this guide assumes that you have already integrated Optimizely feature flags into your code.
+| timestamp | user_id | experiment | variation |
+| --------- | ------- | ---------- | --------- |
+| 2021-06-22T17:35:12.000Z | 165740867980881574 | adding_BNPL_experiment | affirm |
 
-1. Find all places in your code where your feature flags are being invoked.
+By default, Optimizely does not make accessible the data which allows Eppo to determine which users were assigned/exposed to any given feature/experiment. To access this data, you have a couple of options:
 
-**Find the SDK that you're using in the [Optimizely SDK documentation](https://docs.developers.optimizely.com/full-stack/v4.0/docs/create-flag-variations#implement-flag-variations) and identify the syntax you're searching for.** 
+- [Pay to Export data from Optimizely to Snowflake](https://docs.developers.optimizely.com/optimizely-data/docs/snowflake-integration)
```

Comment:
What if someone is not using Snowflake?

Reply:
Good call out. Also that link now redirects to their docs homepage. Will also fix this in a subsequent PR